app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+
+    validates :email, uniqueness: true, presence: true
+    validates_presence_of :name,
+                          :address,
+                          :city,
+                          :state,
+                          :zip
+
+    validates_presence_of :password, require: true
+
+    has_secure_password
```

Comment:
This is great, we will need to attach the hashing algorithm for passwords later down the road.